CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+### Change Log
+
+All notable changes to this project will be documented in this file.
+
+### Sprint 66 ending 2017-08-07
+- Force array context for VMs vnets and images [(#13)](https://github.com/ManageIQ/manageiq-providers-scvmm/pull/13)
```

Comment:
John, do we normally break out the changelog into bugs and enhancements?  If so, this one fixes this bug: https://bugzilla.redhat.com/show_bug.cgi?id=1474404 which will be clearer to a user than the description of this PR.